executor/partition_table_test.go

```diff
@@ -1637,28 +1637,28 @@ func TestPartitionTableWithDifferentJoin(t *testing.T) {
 	// Currently don't support index merge join on two partition tables. Only test warning.
 	queryHash = fmt.Sprintf("select /*+ inl_merge_join(trange, trange2) */ * from trange, trange2 where trange.a=trange2.a and trange.a > %v;", x1)
 	// queryRegular = fmt.Sprintf("select /*+ inl_merge_join(tregular2, tregular4) */ * from tregular2, tregular4 where tregular2.a=tregular4.a and tregular2.a > %v;", x1)
-	// require.True(t,tk.HasPlan(queryHash, "IndexMergeJoin"))
+	// require.True(t,tk.MustHavePlan(queryHash, "IndexMergeJoin"))
```

Comment:
I assume you have a script updating your files, and it missed these ones that are commented out?

I would prefer that also commented out code would be fully updated.